src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -88,6 +89,19 @@ public class AsymmetricSignatureProvider : SignatureProvider
         };
 
         /// <summary>
+        /// Mapping from algorithm to <see cref="ECDsa"/>.KeySize.
+        /// </summary>
+        public static readonly Dictionary<string, int> DefaultECDsaKeySizeInBitsMap = new Dictionary<string, int>()
```

Comment:
I'm not sure to understand what's the point of having such a field public or making it configurable (the key size can't be changed, it's fixed). What will happen if I mess up with the values?